### PR TITLE
#131 - changed PrepareReleaseStrategy class to an interface

### PR DESCRIPTION
**Pull request by hsheikhali1**
Resolves: #131 

## How to Test

1. Run `git checkout master && git pull`
2. Once you have the latest and the greatest; run `git checkout -b hsheikhali1-hsheikhali/131-change-class-to-interface`
3. Run `git pull https://github.com/hsheikhali1/yaclt.git hsheikhali/118-move-package-to-dev-deps`
4. run `yarn` to install dependencies
5. run `yarn build && ./dist/index.js new --changeType "IMPROVED"` to see if `yaclt` can still generate new changelog entries.

### CODE REVIEW

**Comment by hsheikhali1** [comment]
@mrjones2014 let me know if I got the test instructions right this time

**Comment by mrjones2014** [comment]
> @mrjones2014 let me know if I got the test instructions right this time

I'm guessing the branch name you listed is wrong?

>hsheikhali/118-move-package-to-dev-deps

**Comment by mrjones2014** [comment]
Also, @hsheikhali1 not a big deal this time, but whatever email address you're using to make these commits is not associated with your GitHub account. Notice how there's no profile link.

CleanShot-2021-09-15-at-07 05 03

File: changelogs/2021-09-14T19-26-47.533-04-00.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[IMPROVED] Change `PrepareReleaseStrategy` class to an interface {#131}
```
Comment (mrjones2014) [inline changelogs/2021-09-14T19-26-47.533-04-00.md:1]:
```suggestion
[IMPROVED] Change `PrepareReleaseStrategy` abstract class to an interface {#131}
```

**Review by mrjones2014 — CHANGES_REQUESTED** [review]
Small nit in the changelog entry.

**Comment by mrjones2014** [comment]
Found the correct branch under the PR title. Tested the build. Looks Good aside from the one comment I made on the changelog entry.

**Comment by mrjones2014** [comment]
@hsheikhali1 this PR has been sitting here a while. Let me know if you need me to bring it over the finish line.

**Comment by hsheikhali1** [comment]
Oh let me take a look when I get home

**Review by mrjones2014 — APPROVED** [review]

**Pull request merged**